.github/workflows/test.yaml

```diff
@@ -137,11 +141,16 @@ jobs:
           make run &
         env:
           UI: 1
-      - name: Run e2e test
-        run: |
-          cd ui/tests
-          yarn
-          yarn test
+          TEST_COMPATIBILITY: "${{ matrix.feature_flag == '5.0.0' && '1' || '0' }}"
```

Comment:
How about using `FEATURE_VERSION: ${{ matrix.feature_version }}`? It seems to be more clear and depends less on the Makefile.